vernac/g_vernac.mlg

```diff
@@ -42,7 +42,7 @@ let search_queries = Entry.create "search_queries"
 let subprf = Entry.create "subprf"
 
 let quoted_attributes = Entry.create "quoted_attributes"
-let class_rawexpr = Entry.create "class_rawexpr"
+let coercion_class = Entry.create "coercion_class"
```

Comment:
```suggestion
let coercion_class = Entry.create "coercion_class"
let class_rawexpr = coercion_class
  [@@deprecated "Deprecated in 8.17; use 'coercion_class' instead"]
```

This breaks serapi.  Maybe deprecate as shown and do an overlay for serapi?

Reply:
I've deprecated it, but I won't create an overlay for serapi. It is up to them to change as per Coq plugin policy. ping @ejgallego anyway.

Reply:
Or nevermind, I wrote an overlay and didn't bother deprecating.

Reply:
The deprecation is for the benefit of plugins that are not in CI.

> It is up to them to change as per Coq plugin policy.

Sounds good on paper, but not realistic unless the plugin writer is very responsive.  Otherwise you end up waiting until the cows come home.